frontend/package.json

```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "start": "next start",
     "build": "next build",
-    "test": "jest",
+    "test": "TZ=utc jest",
```

Comment:
question: why are we forcing this timezone for all tests? Isn't it better to use something like `timezone-mock`?

Reply:
The issue is that being `Timezone.Local` the default option if not timezone is provided, tests were passing on local environment and failing on the CI (due to different locales).

I am open to suggestions. Going to check `timezone-mock`.

Reply:
Yes, I know the issue and why you added `TZ=utc`, I just think it's better to mock it case by case than globally